### ISSUE TITLE
Performance benchmarks

### ISSUE THREAD

**Issue by mjskay** [stan-dev/posterior#159]
Pinging off of https://github.com/stan-dev/posterior/pull/157#issuecomment-846611937, would it be useful to have some standard benchmarks we use for evaluating performance?

Does anyone have experience setting something like this up? I noticed that the [bench](https://github.com/r-lib/bench) package looks to be developing infrastructure for doing this with continuous integration using scripts with calls to `bench::mark()` in a `bench/` subdirectory. Might be worth playing with.

**Comment by jgabry** [comment]
Good idea. I haven't used bench but at first glance I agree it seems worth playing with. Could be a good solution.

**Comment by mjskay** [comment]
Noting here so we don't forget: from the conversation over at https://github.com/stan-dev/posterior/pull/160#issuecomment-847435468 I think that [touchstone](https://lorenzwalthert.github.io/touchstone/index.html), not bench, is the more promising direction for this.

**Comment by assignUser** [comment]
Hey @mjskay! As discussed {touchstone} is now in a stable state, would love your feedback!
